decision_support/src/metrocs/DecisionCriteriaUI.java

```diff
@@ -0,0 +1,52 @@
+package metrocs;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class DecisionCriteriaUI {
+    private final List<String> criteriaList;
+
+    public DecisionCriteriaUI() {
+        criteriaList = new ArrayList<>();
+    }
+
+    // Method to add a new criterion to the list
+    public void addCriteria(String criteriaName) {
+        criteriaList.add(criteriaName);
+    }
+
+    // Method to get all criteria
+    public List<String> getCriteriaList() {
+        return criteriaList;
+    }
+
+    // Method to prompt user for criteria entry
+    public void promptForCriteria() {
+        Scanner scanner = new Scanner(System.in);
+        String input;
+
+        System.out.println("Enter decision criteria. Type 'done' when you are finished:");
+
+        while (true) {
+            System.out.print("Enter criterion: ");
+            input = scanner.nextLine();
+
+            if (input.equalsIgnoreCase("done")) {
+                break;
```

Comment:
Be careful in this case, where the input might never be "done." We also need to provide that option so the user knows what the breaking case is